screens/CreateGamePage.js

```diff
@@ -94,10 +94,18 @@ export class CreateScreen extends React.Component {
         </View>
         <View style = {{ flexDirection: 'row', alignItems: 'center', justifyContent: 'space-around', paddingTop: Dimensions.get('window').height / 30 }}>
         <View style = {{ flexDirection: 'row', alignItems: 'center', justifyContent: 'space-around', paddingTop: 10 }}>
-          <Text style = { this.state.fontLoaded ? styles.cancelText : styles.cancelTextElse }>Cancel</Text>
-          <View style = { styles.postButton }>
-            <Text style = { this.state.fontLoaded ? styles.postButtonText : styles.postButtonTextElse }>Post!</Text>
-          </View>
+          <TouchableOpacity onPress={() =>
```

Comment:
The spacing/newline situation here is odd. Make sure that this line is reduced as much as possible